### ISSUE TITLE
Adjust repo/project layout

### ISSUE THREAD

**Issue by jviereck** [VirtualCSS/planning#3]
_tldr: Use this issue to speak up in case you dislike any of the repo setup choices I have made. If you don't want to do so publically, feel free to reach out to me via any social channel or via email me at (myFirstName) dot (myLastName) at (gmail) dot (com)._

Please use this issue if you want to adjust the way this project is setup. I tired to make good guesses what is the bare minimum (license file, describe rough goal, provide initial issue to express interest in the project) setup for this project should look like but I am open to any suggestion that someone wants to make in adjusting this bare minimum setup. 

If you don't want to communicate publically, feel free to reach out to me via any social channel or via email at (myFirstName) dot (myLastName) at (gmail) dot (com).

Thanks,
- Julian
